src/test/java/org/springframework/data/couchbase/repository/ReactiveCouchbaseRepositoryQueryIntegrationTests.java

```diff
@@ -124,6 +143,23 @@ void count() {
 				airportRepository.save(airport).block();
 			}
 
+			int page = 0;
+			for (; page < 10;) {
```

Comment:
a while(page < 10) is probably better here

Reply:
I've flattened out the loop to better verify the results.